python/paddle/fluid/tests/unittests/test_auto_parallel_dist_tensor_spec.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) 2023 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
wrong path for this file, should be: test/auto_parallel/

Reply:
Done